bot/exts/holidays/halloween/spookynamerate.py

```diff
@@ -261,10 +260,9 @@ async def announce_name(self) -> None:
             winners = []
             for i, winner in enumerate(winner_messages):
                 winners.append(winner)
-                if len(winner_messages) > i + 1:
-                    if winner_messages[i + 1][1]["score"] != winner[1]["score"]:
-                        break
-                elif len(winner_messages) == (i + 1) + 1:  # The next element is the last element
+                if len(winner_messages) > i + 1 and winner_messages[i + 1][1]["score"] != winner[1]["score"]:
+                    break
+                if len(winner_messages) == (i + 1) + 1:  # The next element is the last element
                     if winner_messages[i + 1][1]["score"] != winner[1]["score"]:
                         break
```

Comment:
```suggestion
                if len(winner_messages) > i + 1 and winner_messages[i + 1][1]["score"] != winner[1]["score"]:
                    break
```
Is the second condition here not impossible to reach given the first? I'm struggling to work out what the original code is doing, and whether the change made is equivalent.

Reply:
Not sure what this is doing either. If the conditional was the other way around it might make sense.

I.E specifically checking for `len(winner_messages) == (i + 1) + 1` and then checking for `if len(winner_messages) > i + 1:` but even then is doesn't make sense, since both result in a break 🤷.

I'll just remove as you suggest

Reply:
https://github.com/python-discord/sir-lancebot/pull/1270/commits/3887b52eeb6443c9b84cb537ebad4b0526efd9f2